terraform/terraform_spec.rb

```diff
@@ -23,6 +23,14 @@
     expect(file("/etc/alpine-release")).to be_file
   end
 
+  it "installs the right version of Alpine" do
+    expect(os_version).to include("Alpine Linux 3.3")
+  end
```

Comment:
> If we stop checking the Alpine version should we also remove the Terraform version check on line 55?

We shouldn't remove it, because we care about the specific version of Terraform and have written code in the `Dockerfile` to use the version in the environment variable. We'd want to know if (hypothetically) someone changed the URL to "latest" (if that exists) or the Terrraform website provided the wrong version.

Colin and I talked a bit about testing the Alpine version. I think it makes more sense to test that it's Alpine (because we want a minimal container and ability to use `apk` commands) rather than the specific version (which we want for pinning but not because of that specific version).

Though there's a separate can of worms about whether we should be pinning the versions of packages within an Alpine series.